presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -43,6 +43,14 @@
         extends PrestoCliLauncher
         implements RequirementsProvider
 {
+    @Inject
+    @Named("databases.presto.server_address")
+    protected String serverAddress;
```

Comment:
It's in launchPrestoCliWithServerArgument(), in the else clause.  So, it is used when databases.presto.cli_kerberos_authentication is false.  Should I use `databases.presto.cli_server_address` there instead?